hls-plugin-api/src/Ide/Types.hs

```diff
@@ -890,9 +893,52 @@ instance GEq IdeNotification where
 instance GCompare IdeNotification where
   gcompare (IdeNotification a) (IdeNotification b) = gcompare a b
 
+-- | Restricted version of 'LspM' specific to plugins
+newtype PluginM config a = PluginM { _runPluginM :: LspM config a }
```

Comment:
Suggest `HandlerM`? since it's the monad that's appropriate for handlers?

Reply:
Renamed